content/docs/2.7/providers/azure-key-vault.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Azure Key Vault secret(s)"
```

Comment:
My suggestion for this and every other title containing "(s)" is to drop the parentheses and simply use the singular form (which seems to match the provider filename in most cases).

```suggestion
title = "Azure Key Vault secret"
```

The intent is that this would read as the _Azure Key Vault secret_ provider.

WDYT @tomkerkhove et all?

Reply:
Fine by me, I don't have a preference

Reply:
Done! 👍